mod.config

```diff
@@ -18,3 +18,48 @@ AUTOMATIC_ENGINE_EXTRACT_DIRECTORY="./engine_temp"
 AUTOMATIC_ENGINE_TEMP_ARCHIVE_NAME="engine.zip"
 
 ENGINE_DIRECTORY="./engine"
+
+# The prefix used for the installer filenames
+# The debian package name will lower-case this, and on all platforms the version tag will be appended
+PACKAGING_INSTALLER_NAME="ExampleMod"
+
+# The URL that the "FAQ" button in the crash dialog takes the user to
+PACKAGING_FAQ_URL="http://wiki.openra.net/FAQ"
+
+# The name to display in the OSX menu bar and "About" window.
+# This is used in:
+#  - Crash dialogs (all platforms)
+#  - macOS menu bar
+#  - macOS "About" window
+#  - Windows installer
+#  - Windows start menu
+#  - Windows desktop shortcut
+#  - Windows "Programs and Features" list
+PACKAGING_DISPLAY_NAME="Example Mod"
+
+# This is used in:
+#  - Windows "Add/Remove Programs" list
+PACKAGING_WEBSITE_URL="http://openra.net"
+
+# This is used in:
+#  - Windows "Add/Remove Programs" list
+PACKAGING_AUTHORS="Example Mod authors"
+
+# The name of the .app bundle that is compiled
+PACKAGING_OSX_APP_NAME="OpenRA - Example Mod.app"
+
+# The git tag to use for the OpenRA OSX Launcher
+PACKAGING_OSX_LAUNCHER_TAG="osx-launcher-20170604"
+
+# The url to download containing the OpenRA OSX launcher files
+PACKAGING_OSX_LAUNCHER_SOURCE="https://github.com/OpenRA/OpenRALauncherOSX/releases/download/${PACKAGING_OSX_LAUNCHER_TAG}/launcher.zip"
+
+# Temporary file name used when downloading the OpenRA OSX launcher files
+PACKAGING_OSX_LAUNCHER_TEMP_ARCHIVE_NAME="launcher.zip"
```

Comment:
I can reorder these into basic and advanced sections, so that all the values that we expect to be modified are separated from the values we expect won't be modified.